### PR TITLE
Use `wit-vue-ui`

### LINKED ISSUE #423: Use wit-vue-ui
Use `wit-vue-ui` component library instead of current ui components

### PR DESCRIPTION
**Pull request by gabaldon**
close #423

### CODE REVIEW

File: components/cards/BuildWithWitnet.vue
Comment (Tommytrg) [inline components/cards/BuildWithWitnet.vue:1]: <table>
<tr>
<th> PR </th>
<th> Production </th>
</tr>
<tr>
<td>


### 
image

</td>
<td>

image

</td>
</tr>
</table>

File: components/sections/CommunitySection.vue
Comment (Tommytrg) [inline components/sections/CommunitySection.vue:1]: <table>
<tr>
<th> PR </th>
<th> Production </th>
</tr>
<tr>
<td>

image

</td>
<td>

image

</td>
</tr>
</table>

File: components/FooterSection.vue
Comment (Tommytrg) [inline components/FooterSection.vue:1]: Add first letter capitalization and also underline links

<table>
<tr>
<th> PR </th>
<th> Production </th>
</tr>
<tr>
<td>

image

</td>
<td>

image

</td>
</tr>
</table>

**Review by Tommytrg — CHANGES_REQUESTED** [review]

File: components/FooterSection.vue
Comment (gabaldon) [reply components/FooterSection.vue:1]: Where should I add first letter capitalization?

**Review by Tommytrg — COMMENTED** [review]
Could we add an space between the date and the first word?
image

**Review by Tommytrg — APPROVED** [review]

**Pull request merged**